### ISSUE TITLE
cd .. doesn't work if previous cd is with "/" at the end

### ISSUE THREAD

**Issue by pbedia-stratio** [mlorenzo-stratio/vcli#1]
```
[200] vcli@vault.service.paas.labs.stratio.com:8200 > cd dcs/
[200] vcli@vault.service.paas.labs.stratio.com:8200/dcs/ > cd ..
[200] vcli@vault.service.paas.labs.stratio.com:8200/dcs > cd ..
[200] vcli@vault.service.paas.labs.stratio.com:8200 > 
```

**Comment by ml0renz0** [comment]
Fixed in 92d77e58454787a7107ca56c361f747cbdd990a9

**Comment by ml0renz0** [comment]
Had some typo. Fixed in this commit de6753d
